docs/timers.md

```diff
@@ -82,4 +82,4 @@ called.
 Sample Apps
 -----------
 * [Timers sample](../samples/Timers.js)
-* [Spaceship2 sample](../arduino/starterkit/Spaceship2.js)
+* [Spaceship2 sample](../samples/Spaceship2.js)
```

Comment:
Good catch, the old path was wrong. But the new path is also wrong. :)

It should be:
`../samples/arduino/starterkit/Spaceship2.js`